### PR TITLE
Memorystore oc

### PR DESCRIPTION
**Pull request by karthitect**
Hello, this is a new community tutorial submission: "Client-side Tracing of Cloud Memorystore for Redis Workloads with OpenCensus"

### CODE REVIEW

**Comment by ToddKopriva** [comment]
[ ] - editorial review - @ToddKopriva 
[ ] - final production review for publishing - @ToddKopriva

File: tutorials/memorystore-oc/index.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+*   Make sure that [billing is enabled for your project](https://cloud.google.com/billing/docs/how-to/modify-project).
+
+### Initialize the environment
+
+1.  Start a [Cloud Shell instance](https://console.cloud.google.com/home/dashboard?cloudshell%3Dtrue).
+
+2.  In Cloud Shell, set the default Compute Engine zone to the zone where you are going to create your Cloud Dataproc clusters. This tutorial uses the us-central1-a zone in the us-central1 region.
+
+```bash
```
[the remark this answers is not in the document]
Comment (karthitect) [reply tutorials/memorystore-oc/index.md]: Ok, thanks

File: tutorials/memorystore-oc/index.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+![](images/image7.png)
+
+Once logged into the VM, run the following command to install the redis-cli, git, the Java 8 JDK and maven:
+
+```bash
+$ sudo apt-get install redis-tools git openjdk-8-jdk maven -y
+```
+
+Run the following command within the trace-client VM to ensure that you can reach the Cloud Memorystore for Redis instance you created earlier:
```
[the remark this answers is not in the document]
Comment (karthitect) [reply tutorials/memorystore-oc/index.md]: Cloud Memorystore for Redis is the full name of the product/service

File: tutorials/memorystore-oc/java/src/test/java/com/example/memorystore/AppTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    /**
+     * @return the suite of tests being tested
+     */
+    public static Test suite()
+    {
+        return new TestSuite( AppTest.class );
+    }
+
+    /**
+     * Rigourous Test :-)
```
[the remark this answers is not in the document]
Comment (karthitect) [reply tutorials/memorystore-oc/java/src/test/java/com/example/memorystore/AppTest.java]: I deleted the test folder

**Comment by karthitect** [comment]
I've checked in changes from your first editing pass. Thanks!

**Comment by ToddKopriva** [comment]
published:
https://cloud.google.com/community/tutorials/memorystore-oc

**Pull request merged**